Lib/shlex.py

```diff
@@ -306,17 +306,23 @@ def split(s, comments=False, posix=True):
 
 
 _find_unsafe = re.compile(r'[^\w@%+=:,./-]', re.ASCII).search
+_find_unsafe_bytes = re.compile(b'[^\w@%+=:,./-]').search
```

Comment:
```suggestion
_find_unsafe_bytes = re.compile(br'[^\w@%+=:,./-]').search
```
This line generates a `SyntaxWarning` that causes test failure in CI where `assert_python_ok` asserts for no output in stderr that has this `SyntaxWarning`. Perhaps use a raw string ?

```
/Users/karthikeyansingaravelan/stuff/python/cpython/Lib/shlex.py:309: SyntaxWarning: invalid escape sequence \w
  _find_unsafe_bytes = re.compile(b'[^\w@%+=:,./-]').search
```